analysed-packages/glib-networking/version-2.72.2/glib-networking-2.72.2-SPDX2TV.spdx

```diff
@@ -0,0 +1,6468 @@
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: glib-networking-2.72.2.tar.xz/glib-networking-2.72.2/tls/tests/files/update-chain-with-new-root.py
+SPDXID: SPDXRef-item544284
+FileChecksum: SHA1: fa751f48ae765c25e90dcd1105cc1a46bbaf323e
+FileChecksum: SHA256: 5c2c58d5d425aeb40edde3ba9563ed6e962addaf446ebb1c56f258a134de6246
+FileChecksum: MD5: 739d7a8fa90232c0a247ca12675f3f40
+LicenseConcluded: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-lgpl-2.1-plus
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: glib-networking-2.72.2.tar.xz/glib-networking-2.72.2/tls/tests/files/update-certificate-test.py
+SPDXID: SPDXRef-item544261
+FileChecksum: SHA1: aab3ed978de6571ea35512f87d5297d0c2a82489
+FileChecksum: SHA256: 45d3c67704ff396bc8b1a9dcf36465a254d9be05f1cb413b251b2046f06e3788
+FileChecksum: MD5: c116a9f333311164f5fa29752792c213
+LicenseConcluded: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-lgpl-2.1-plus
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: glib-networking-2.72.2.tar.xz/glib-networking-2.72.2/tls/tests/pkcs11/pkcs11t.h
+SPDXID: SPDXRef-item544323
+FileChecksum: SHA1: e29bb7cbf0eac413f7a870fa6b1d2b06e7a72287
+FileChecksum: SHA256: 5824e4b4b8d9078c85f27b0616342c09ce1be084bf0f8e3818f08353d855ad17
+FileChecksum: MD5: 993d51c4201f279e3bf6d640de3d3cc7
+LicenseConcluded: LicenseRef-MPL-2.0 AND LicenseRef-RSA-Cryptoki
+LicenseInfoInFile: LicenseRef-MPL-2.0
+LicenseInfoInFile: LicenseRef-RSA-Cryptoki
+LicenseInfoInFile: LicenseRef-rsa-1990
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: glib-networking-2.72.2.tar.xz/glib-networking-2.72.2/tls/tests/pkcs11/pkcs11.h
+SPDXID: SPDXRef-item544322
+FileChecksum: SHA1: 2257f89036b45193da66778f625ef755eaf183bc
+FileChecksum: SHA256: 193caec0c07c95a1da30994210c98318690669c4d0fa970cbd05be905e02f937
+FileChecksum: MD5: 75074170cb907b8b3c2b8b14e9103d5b
+LicenseConcluded: LicenseRef-MPL-2.0
+LicenseInfoInFile: LicenseRef-MPL-2.0
+FileCopyrightText: <text> Copyright (C) 1994-1999 RSA Security Inc.  </text>
+
+##File
+
+FileName: glib-networking-2.72.2.tar.xz/glib-networking-2.72.2/tls/tests/pkcs11/pkcs11u.h
+SPDXID: SPDXRef-item544324
+FileChecksum: SHA1: 0037c1f22ae7aafa0933120e79d4b466e54dd073
+FileChecksum: SHA256: 2c3a0c44986433060e667e24387c826ef52637d1a3995b9717d66c150d76d4a0
+FileChecksum: MD5: b103f4f2780600f8c8022db373a29917
+LicenseConcluded: LicenseRef-MPL-2.0
+LicenseInfoInFile: LicenseRef-MPL-2.0
+FileCopyrightText: <text> Copyright (C) 1994-1999 RSA Security Inc.  </text>
+
+##File
+
+FileName: glib-networking-2.72.2.tar.xz/glib-networking-2.72.2/tls/tests/pkcs11/pkcs11p.h
+SPDXID: SPDXRef-item544321
+FileChecksum: SHA1: 4c5976dbfa8d4d6728c381aed95334b751cfb09d
+FileChecksum: SHA256: f83e09177de2e19f75c2aa76039d19b4fdb36488cf6e7dffbdafa618e9cac3fa
+FileChecksum: MD5: 70d9a4f154973e7c6c93f10cfa295647
+LicenseConcluded: LicenseRef-MPL-2.0
```

Comment:
Below the MPL-2.0 header there is the following information:
Licence to copy this document is granted provided that it is identified as "RSA Security Inc. Public-Key Cryptography Standards (PKCS)" in all material mentioning or referencing this document.

This Permission-Notice is missing as concluded license. Additionally it requires an acknowledgment, which is also missing:

This software contains RSA Security Inc Public-Key Cryptography Standards (PKCS)